tests/test_automatic_fields.py

```diff
@@ -72,7 +79,15 @@ def automatic_test(self, conn_id, testable_streams):
                 # verify that only the automatic fields are sent to the target
                 self.assertEqual(
                     actual_fields_by_stream.get(stream, set()),
-                    self.expected_primary_keys().get(stream, set()) |
+                    expected_primary_keys |
```

Comment:
Why was this changed to a variable, but not `self.expected_replication_keys().get(stream, set())`?

Reply:
We have created a variable named `expected_primary_keys` which contains the same value. Hence, updated the code to use the same variable here.